try {
throw new Error("There is something wrong")    
}
catch(error){
    console.log(error.message)
}
